rdbms/src/main/java/org/spine3/server/storage/jdbc/command/query/CommandStorageQueryFactory.java

```diff
@@ -59,7 +62,7 @@ public void setLogger(Logger logger) {
     }
```

Comment:
Is the logger optional in this case? If it isn't, pls inject it via constructor.